pyvista/core/pointset.py

```diff
@@ -1248,7 +1257,9 @@ def _from_cells_dict(self, cells_dict, points, deep=True):
             cell_types, cells, offset = create_mixed_cells(cells_dict, nr_points)
             self._from_arrays(offset, cells, cell_types, points, deep=deep)
 
-    def _from_arrays(self, offset, cells, cell_type, points, deep=True):
+    def _from_arrays(
+            self, offset, cells, cell_type, points, deep=True, force_float=True
```

Comment:
```suggestion
            self, offset, cells, cell_type, points, deep=True, force_float=True,
```
(Just to reduce the diff when the next kwarg is added on the next line)